examples/interoperability/app.js

```diff
@@ -0,0 +1,125 @@
+import 'babel-polyfill';
+/* global document, window,*/
+/* eslint-disable no-console */
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {
+  ScatterplotLayer, OrthographicViewport, COORDINATE_SYSTEM
+} from '../../src';
```

Comment:
Yes, I'm aware. There're some changes and one commit after rc.1 and I wanted to test the example before I publish the rc.2. Once those changes are landed, I will change back import from '../../src' to 'deck.gl'.

This is also the reason for the error that @shaojingli saw. We are having two copies of react and I had to npm link it. will be resolved after we switch to import from 'deck.gl'